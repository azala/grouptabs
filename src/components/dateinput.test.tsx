import React from "react";
import { render } from "@testing-library/react";
import DateInput from "./dateinput";

it("renders prefilled input", () => {
  const { container } = render(
    <DateInput date="2020-03-22" onChange={jest.fn()} />
  );
  expect(container.firstChild).toMatchSnapshot();
});
