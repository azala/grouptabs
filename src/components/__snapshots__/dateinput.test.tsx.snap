// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders prefilled input 1`] = `
<div
  class="form-row-input date-input"
>
  <button
    class=""
    type="button"
  >
    today
  </button>
  <button
    class=""
    type="button"
  >
    yesterday
  </button>
  <input
    type="date"
    value="2020-03-22"
  />
</div>
`;
